packages/mds-daily/api.ts

```diff
@@ -53,20 +52,10 @@ async function agencyMiddleware(req: DailyApiRequest, res: DailyApiResponse, nex
         }
 
         if (provider_id) {
-          if (!isUUID(provider_id)) {
-            await log.warn(req.originalUrl, 'bogus provider_id', provider_id)
-            return res.status(400).send({
-              result: `invalid provider_id ${provider_id} is not a UUID`
-            })
+          const providerIdResult = await providerClaimMiddleware(req, res)
+          if (providerIdResult === false) {
+            return
```

Comment:
Don't do this ... add `providerClaimMiddleware` to the route (before the request handler) ... if you _get_ to the handler then you can get `provider_id` from `res.locals.provider_id`